docs/scripts.rst

```diff
@@ -130,6 +130,10 @@ Explanation for the keys:
 - ``urls``: dictionary of table names and the respective urls
 - ``version``: "1.0.0"
 
+``fields``: Although retriever automatically detects the schema of the table,
```

Comment:
We could change to 
```
fields: Although the Data retriever automatically detects the schema of the table,
it is recommended to describe the schema of the table.
This enables processing of the data faster since column detection increases the processing time. 
```

Feel free to add something like this. You may add it here or down after the heading: **Full control over column names and data types**

The ``fields`` enables full control of the columns, this includes, renaming columns, skipping unwanted columns and combining columns. 
For example, given data with columns,``a,b,c`` a user can pickle out `b,c` by using the skip datatype for column a.

```
                            "schema": {
                                "fields": [
                                    {
                                        "name": "a",
                                        "type": "skip"
                                    },
                                    {
                                        "name": "b",
                                        "type": "int"
                                    },
                                    {
                                        "name": "c",
                                        "type": "int"
                                    }
                                ]
```

Each value of the field has a ``name``  and a ``type`` .
The name represents the column name in the data. 
The type represents the type of data.
```
pk-auto: Auto generated primary key starting from 1
pk-[char,int,double]: primary key with data type
char: strings
int: integers
double:floats/decimals
ct-[int,double,char]:Cross tab data
```